app/assets/stylesheets/global.css.scss

```diff
@@ -0,0 +1,50 @@
+#gradient {
+  background: red; /* For browsers that do not support gradients */
+  /* For Safari 5.1 to 6.0 */
+  background: -webkit-linear-gradient(left, #007EEE, #0095F3, #007FED);
+  /* For Opera 11.1 to 12.0 */
+  background: -o-linear-gradient(left, #007EEE, #0095F3, #007FED);
+  /* For Fx 3.6 to 15 */
+  background: -moz-linear-gradient(left, #007EEE, #0095F3, #007FED);
+  /* Standard syntax */
+  background: linear-gradient(to right, #007EEE, #0095F3, #007FED);
+}
+
+.top-bar a {
+  background-color: rgba(0, 163, 255, .0)
+}
+
+.top-bar ul {
+  background-color: rgba(0, 163, 255, .0)
+}
+
+.top-bar li {
+  background-color: rgba(0, 163, 255, .0)
+}
+
+.border {
+  height: .2rem;
+  background-color: #69F1FE;
+}
+button.button a {
+  color: white;
+}
+
+.chart {
+  margin: 3rem 0rem;
+}
+
+.text-center button {
+  margin: 2rem;
+}
+
+.flash-notice {
+}
+
+.actions input {
+  background-color: #1583cc;
+  padding: .2rem 1.2rem;
```

Comment:
`.2` should be written with a leading zero as `0.2`<br>Properties should be ordered background-color, border-radius, border-style, color, padding